cmd/rpcdaemon/commands/eth_receipts.go

```diff
@@ -275,11 +275,28 @@ func (api *APIImpl) GetTransactionReceipt(ctx context.Context, txnHash common.Ha
 	var ok bool
 
 	blockNum, ok, err = api.txnLookup(ctx, tx, txnHash)
-	if !ok || blockNum == 0 {
-		// It is not an ideal solution (ideal solution requires extending TxnLookupReply proto type to include bool flag indicating absense of result),
-		// but 0 block number is used here to mean that the transaction is not found
-		return nil, nil
+
+	if !ok {
+		if cc.Bor == nil {
+			return nil, nil
+		}
+
+		blockNum = api._blockReader.HeaderByHash(ctx, tx, txnHash)
```

Comment:
This is not working, I think this needs hash of header(block), Not the txn.

Reply:
yeah I think my Pr is pretty much useless I'll close it.